model/ProctorioApiService.php

```diff
@@ -95,7 +95,10 @@ public function setProctorioUrlLibraryService(ProctorioService $proctorioUrlLibr
     private function requestProctorioUrls(DeliveryExecutionInterface $deliveryExecution): string
     {
         $proctorioService = $this->getProctorioLibraryService();
-        $launchUrl = $this->getLaunchService()->generateUrl($deliveryExecution->getIdentifier());
+        $launchUrl = $this->getLaunchService()->generateUrl(
+            $this->getLaunchService()::URI_PARAM_EXECUTION,
```

Comment:
```suggestion
            LaunchService::URI_PARAM_EXECUTION,
```
- Since this is a constant and it does not rely on object state we should avoid call it based on instance.
- Also It is better to read, you know exactly from which class this constant belongs.